docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,51 @@ title: Build a plugin
 weight: 200
 ---
 
-# Build a plugin
+# Welcome to the Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing all of your data through beautiful, flexible dashboards. Plugins extend its built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
 
-## Get started
+Many plugins are available in the [Catalog](https://grafana.com/grafana/plugins/) and others are built by developers for private use or to contribute them to the open source community. 
 
-The easiest way to start developing Grafana plugins is to use the Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin).
+If you are a Grafana plugin developer or want to become one, this documentation is for you. Use it to get productive quickly, follow along with step-by-step tutorials, or you can refer back to it as you develop.  
 
-Open the terminal, and run the following command in your [plugin directory]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}):
+## Plugin basics
 
-```bash
-npx @grafana/create-plugin@latest
-```
+These are among the types of plugins you can create:
 
-Follow the questions and you will have a starter plugin ready to develop.
+- **Panel plugins** - Visualize data and navigating between dashboards.
```

Comment:
can we add links to the tutorials to create these kind of plugins?

Reply:
Let's hold off. This section is probably going to be moving to a separate topic on types of plugins soon. As well, we don't have tutorials for all the types in this list at this time.